python/pyspark/mllib/recommendation.py

```diff
@@ -61,7 +61,7 @@ class MatrixFactorizationModel(JavaModelWrapper, JavaSaveable, JavaLoader):
 
     >>> model = ALS.train(ratings, 4, seed=10)
     >>> model.userFeatures().collect()
-    [(1, array('d', [...])), (2, array('d', [...]))]
+    [(1, DenseVector([...])), (2, DenseVector([...]))]
```

Comment:
We shouldn't change the return type. I'm not sure `DenseVector` is a safe replacement for array.